### PR TITLE
Create file2.md

### PR DESCRIPTION
**Pull request by mohritaroh**

### CODE REVIEW

File: file2.md [hunk 1]
```diff
@@ -0,0 +1 @@
+this is anothor file
```
Comment (mohritaroh2) [inline file2.md]: wite more

**Review by mohritaroh2 — COMMENTED** [review]
rewite now

File: file2.md (hunk 1 above)
Comment (mohritaroh) [reply file2.md]: wrote!

**Review by mohritaroh — COMMENTED** [review]
wrote

File: file2.md (hunk 1 above)
Comment (mohritaroh) [reply file2.md]: rewrite this line

**Review by mohritaroh — COMMENTED** [review]
rewite!

**Pull request merged**